README.md

```diff
@@ -6,6 +6,39 @@ Checks the official JSON files every once in a while (configurable in the config
 file) for changes and sends new warnings to registered users based on the
 coordinates the entered.
 
+## Install
+
+(tested on Linux Mint 20.2)
+
+1. Clone the repository
+
+`git clone https://github.com/jplitza/nina_xmpp.git`
+
+2. Install dependencies
+
+```
+sudo pip3 install aioxmpp httpx shapely geoalchemy2 argparse_logging
+sudo apt-get install libsqlite3-mod-spatialite
+```
+
+3. Build
+
+`python3 setup.py build`
+
+4. Install
+
+`sudo python3 setup.py install`
```

Comment:
Instead of `sudo` I suggest to say `#` (for security purposes und for users who use other means of priviledge escalation).

Reply:
For a user installation which does not require root permissions one could alternatively use

```
python3 setup.py develop --user
```

Reply:
Valid point.